source/MRViewer/ImGuiHelpers.h

```diff
@@ -139,6 +139,14 @@ MRVIEWER_API void PlotCustomHistogram( const char* str_id,
 
 /// begin typical state plugin window
 MRVIEWER_API bool BeginStatePlugin( const char* label, bool* open, float width );
+
+/// begin state plugin window with custom style.  If you use this function, you must call EndCustomStatePlugin to close the plugin correctly.
```

Comment:
as far as you start comment with small latter, better don't start new sentence with capital one:
```c++
/// begin state plugin window with custom style, if you use this function, you must call EndCustomStatePlugin to close the plugin correctly